gensim/models/fasttext.py

```diff
@@ -1059,7 +1059,10 @@ def __contains__(self, word):
             False
 
         """
-        return True
+        if self.bucket == 0:  # check for the case when char ngrams not used
```

Comment:
The above doc-comment 'note' should be updated to reflect this new behavior & its rationale.

Reply:
@rock420 how about:

```
This method **always** returns True with char ngrams, because of the way FastText works.
```

Reply:
Updated the doc-comment